tests/quickcheck.rs

```diff
@@ -453,3 +457,194 @@ fn graph_sccs() {
     }
     quickcheck::quickcheck(prop as fn(_) -> bool);
 }
+
+#[derive(Debug, Clone)]
+struct DAG<N: Default + Clone + Send + 'static>(Graph<N, ()>);
+
+impl<N: Default + Clone + Send + 'static> quickcheck::Arbitrary for DAG<N> {
```

Comment:
We don't need to fix it now, but N should probably just be Arbitrary and generated by qc.